extensions/warp-pd-flatfile/examples/flatfile-cache.rs

```diff
@@ -26,7 +25,17 @@ fn main() -> anyhow::Result<()> {
         DimensionData::from("Cargo.toml"),
     )?;
 
+    let data_account = Sata::default().encode(warp::libipld::IpldCodec::DagJson, warp::sata::Kind::Reference, b"Hi Account");
+    let data_messaging = Sata::default().encode(warp::libipld::IpldCodec::DagJson, warp::sata::Kind::Reference, b"Hi Messaging");
+    let data_http = Sata::default().encode(warp::libipld::IpldCodec::DagJson, warp::sata::Kind::Reference, b"Hi Http");
+    let data_export = Sata::default().encode(warp::libipld::IpldCodec::DagJson, warp::sata::Kind::Reference, b"Hi Data Export");
+    
+
     storage.add_data(DataType::from(Module::FileSystem), &data)?;
+    storage.add_data(DataType::from(Module::Accounts), &data_account.unwrap())?;
+    storage.add_data(DataType::from(Module::Messaging), &data_messaging.unwrap())?;
+    storage.add_data(DataType::from("http"), &data_http.unwrap())?;
```

Comment:
You could use `DataType::Http` or `DataType::DataExport` but this is fine as well.